### PR TITLE
only assign kcode if ruby version is < 1.9.0

### PR DESCRIPTION
**Pull request by ashleygwilliams**
solves #168

### CODE REVIEW

**Comment by ashleygwilliams** [comment]
@TrevorBramble @zzak review plz <3

File: lib/sinatra/respond_with.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 require 'sinatra/json'
 require 'sinatra/base'
 
-$KCODE = "UTF-8"
+$KCODE = "UTF-8" unless RUBY_VERSION > "1.9.0"
```
Comment (TrevorBramble) [inline lib/sinatra/respond_with.rb]: Not `>=`?

Comment (zzak) [inline lib/sinatra/respond_with.rb]: This works, also we don't support 1.9.0, so it's probably fine.

Alternatively we could do: `RUBY_VERSION.to_f >= 1.9`

**Comment by ashleygwilliams** [comment]
no, 1.9.0 introduced warnings for kcode, doesn't need it @TrevorBramble (as far as i can tell)

**Comment by ashleygwilliams** [comment]
http://stackoverflow.com/questions/21574/what-is-the-difference-between-ruby-1-8-and-ruby-1-9

File: lib/sinatra/respond_with.rb (hunk 1 above)
Comment (TrevorBramble) [inline lib/sinatra/respond_with.rb]: Between that and https://github.com/sinatra/sinatra-contrib/pull/169#issuecomment-110176938 :+1:

**Comment by TrevorBramble** [comment]
Weigh :anchor: and :ship: away!

**Comment by ashleygwilliams** [comment]
:dancer:

**Comment by zzak** [comment]
Thank you @ashleygwilliams!

**Pull request merged**